src/server/api/controllers/users.controller.js

```diff
@@ -43,8 +58,8 @@ const getUserFavorites = async (user_id) => {
     return error.message;
```

Comment:
If you feel like it, you can add the same error handling here since you don't want to just return the error message here.